packages/app/src/app/index.js

```diff
@@ -94,10 +97,7 @@ if (process.env.NODE_ENV === 'production') {
   ];
 
   try {
-    initializeAnalytics({
-      sentryDSN: window._env_?.SENTRY_DSN,
-      amplitudeApiKey: window._env_?.AMPLITUDE_API_KEY,
-    });
+    initializeSentry(window._env_?.SENTRY_DSN);
```

Comment:
decoupled sentry and amplitude here, as only the latter is conditioned by the cookie consent

Reply:
I didn't get how you enable Sentry only after the cookie consent, because I could only see amplitude being enabled after that